metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Distribute and enable Chef Exception and Report handlers'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.1.1'
+version '2.2.0'
```

Comment:
can you remove this from the commit? we do metadata updates when we do the release. Thanks!